docs/syntax.rst

```diff
@@ -132,12 +132,20 @@ Supported scalar functions:
 * ``ti.tanh(x)``
 * ``ti.exp(x)``
 * ``ti.log(x)``
+* ``ti.exp2(x)``
+* ``ti.log2(x)``
+* ``ti.step(x)``
+* ``ti.smoothstep(x)``
 * ``ti.random(type)``
-* ``abs(x)``
 * ``max(a, b)``
 * ``min(a, b)``
-* ``ti.length(dynamic_snode)``
+* ``abs(x)``
+* ``ti.sign(x)``
+* ``ti.clamp(x, a, b)``
+* ``ti.mix(x, y, a)``
+* ``ti.fract(x)``
 * ``x ** y``
+* ``ti.sizeof(dynamic_snode)``
```

Comment:
I think `sizeof` here can be slightly confusing for people from C-family languages since it is a compile-time operator that returns a compile-time constant.

Reply:
How about snode.size()? It can also apply to static nodes for generic-programming-support.